ptx/sec_deriv_interpret.ptx

```diff
@@ -341,7 +341,9 @@
           <!-- START figures/fig_xsquared.tex -->
           <image xml:id="img_xsquared" width="47%">
           <shortdescription>a graph of f(x)=x^2<shortdescription>
-            <description><p>the curve is parabolic with the focus being at the origin. As the x values get bigger and bigger the curve gers further from the y axis on both sides as both the ends approach infinity. Buth the rate at which
+            <description><p>the curve is parabolic with the focus being at the origin. 
+            As the x values get bigger and bigger the curve gers further from the y axis
```

Comment:
In the long description, all variables need to be in math tags. `<m>x</m>`, etc.

Reply:
@cr1nja I don't see any of the changes requested. x and y still need math tags. Also, the first letter of the paragraph should be capitalized.